scripts/upgrade

```diff
@@ -12,18 +12,30 @@ source /usr/share/yunohost/helpers
 #=================================================
 # LOAD SETTINGS
 #=================================================
+ynh_print_info "Load settings"
 
 app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get $app domain)
 path_url=$(ynh_app_setting_get $app path)
 is_public=$(ynh_app_setting_get $app is_public)
 final_path=$(ynh_app_setting_get $app final_path)
+language=$(ynh_app_setting_get $app language)
 db_name=$(ynh_app_setting_get $app db_name)
 
+lang=$(ynh_app_setting_get $app lang)
+
+if [ -n "$lang" ]
+then
+    language="$lang"
+    ynh_app_setting_set "$app" language "$language"
+	ynh_app_setting_delete "$app" lang
+fi
```

Comment:
Not sure that it's really needed to replace `lang` by `language`, but anyway, this should be placed into `ENSURE DOWNWARD COMPATIBILITY` part

Reply:
I made the change for two reasons:

1. The lang argument was a `"choices": ["English", "Francais"],` translated in the install script to
![capture](https://user-images.githubusercontent.com/30271971/52692910-7d2fd580-2f65-11e9-839c-482c87bdbca8.PNG)

2. To make the package maintenance easier in the future regarding example_ynh

Reply:
Indeed, removing that part is a good idea, it's better to have real value directly, and en, fr is quite clear.
My question was more, is it really useful to change `$lang` by `$language` ?

Reply:
Not usefull at all, just to comply with example_ynh.
If you agree, I will rollback to $lang ?

Reply:
The example app is just an example for packagers to start a new package.
It's important to copy the new features or the modifications of the template itself. But that's just an example.
So you don't need to use `$language` because example_ynh use it, as well as it's not needed to copy comments that used only to explain how to build the package.
For all the other changes in this PR, that's a very good work.

Reply:
Ok then, I will do the changes. Is it better to do them in the branch I was working for this pull request or to create a new branch and make a new pull request ?

Reply:
The best thing to do is to push directly into this PR, so it will resolve discussion on this thread.